playbooks/roles/airship-setup-deployer/tasks/main.yml

```diff
@@ -116,3 +116,8 @@
   template:
     src: clouds.yml.j2
     dest: /etc/openstack/clouds.yaml
+
+- name: Save Shipyard env variables to ~/.profile
+  template:
+    src: profile.j2
+    dest: "{{ ansible_user_dir }}/.profile"
```

Comment:
Do we really want to override a user's profile?
That's very intrusive.

Are you sure you don't want to ensure those lines are part of the profile instead?
For that, you can use "lineinfile" module.

Reply:
Yes, good point, I've actually realized yesterday as well. I'll change it